ouroboros-consensus-cardano/tools/db-analyser/Block/Cardano.hs

```diff
@@ -81,14 +85,20 @@ parseCardanoArgs :: Parser CardanoBlockArgs
 parseCardanoArgs = CardanoBlockArgs
     <$> argsParser Proxy
     <*> argsParser Proxy
+    <*> strOption (mconcat [
+            long "configAlonzo"
+          , help "Path to config file"
+          , metavar "PATH"
+          ])
 
 mkCardanoProtocolInfo ::
      Byron.Genesis.Config
   -> Maybe PBftSignatureThreshold
   -> ShelleyGenesis StandardShelley
+  -> SL.AlonzoGenesis
```

Comment:
Same as in `CanHardFork.hs`. I would call this `Core.TranslationContext (AlonzoEra c)` or (not sure if this would blow things up) say `Core.TranslationContext (AlonzoEra c) ~ SL.AlonzoGenesis` to make explicit that we are relying on that equality to hold.

Reply:
Replied at https://github.com/input-output-hk/ouroboros-network/pull/3138#discussion_r632438113